### PR TITLE
Preserve form data

### PR DESCRIPTION
**Pull request by aimalamiri**
## Store Contact form data in local storage
1. Storing form data in local storage
2. Getting form data from storage and inserting it into the form when the page loads.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -212,6 +219,12 @@ window.addEventListener('load', () => {
       projectModal.style.top = `${window.pageYOffset}px`;
     });
   }
+
+  // Insert Form data from formData object into contact form
+  formData = JSON.parse(localStorage.getItem('formData'));
+  for (let i = 0; i < inputs.length; i += 1) {
+    inputs[i].value = formData[inputs[i].name];
+  }
```
Comment (thecodechaser) [inline script.js]: - [x] Your form data is not being submitted to local storage there is a problem with your functions, Please make sure the local storage functionality works fine.

### See the image of the error.

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @aimalamiri👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (aimalamiri) [reply script.js]: Oh yeah 😊 I made a mistake when I was trying to fix the linter error.

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @aimalamiri & @kinginthenorthcodez , 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Descriptive readme file.
:heavy_check_mark: Well implemented requested changes..

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**